src/manifolds/Tucker.jl

```diff
@@ -0,0 +1,876 @@
+
+@doc raw"""
+Tucker{N, R, D, 𝔽} <: AbstractManifold{𝔽}
```

Comment:
```suggestion
    Tucker{N, R, D, 𝔽} <: AbstractManifold{𝔽}
```
The first line should be indented by 4 spaces to be rendered as code.